src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Fixtures/php/translator_globals.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+$container->loadFromExtension('framework', [
+    'annotations' => false,
+    'http_method_override' => false,
+    'handle_all_throwables' => true,
+    'php_errors' => ['log' => true],
+    'translator' => [
+        'globals' => [
+            '%%app_name%%' => 'My application',
+            '{app_version}' => '1.2.3',
+            'url' => ['message' => 'url', 'parameters' => ['scheme' => 'https://'], 'domain' => 'global'],
```

Comment:
even in tests, the key should use delimiters to be clear about the fact it's going to be needed in real apps also